endpoints.yml

```diff
@@ -0,0 +1,1520 @@
+  - /cloud/cloud-networks/f84c058d-9343-4881-96a2-e87057a76954/changelog/
+  cloud:cloudnetwork_list:
+  - /cloud/cloud-networks/
+  - /cloud/cloud-networks/?per_page=4&page=5
+  cloud:cloudnetwork_notes:
+  - /cloud/cloud-networks/48f6ca3e-d9b1-4634-8f5f-3166c62a66ab/notes/
+  - /cloud/cloud-networks/f84c058d-9343-4881-96a2-e87057a76954/notes/
+  cloud:cloudresourcetype:
+  - /cloud/cloud-resource-types/de3a66de-bf98-4f53-89e9-4be79ee0632d/
+  - /cloud/cloud-resource-types/17a82f09-4da4-4543-a3be-7a4678bd694a/
+  cloud:cloudresourcetype_changelog:
+  - /cloud/cloud-resource-types/de3a66de-bf98-4f53-89e9-4be79ee0632d/changelog/
+  - /cloud/cloud-resource-types/17a82f09-4da4-4543-a3be-7a4678bd694a/changelog/
+  cloud:cloudresourcetype_list:
+  - /cloud/cloud-resource-types/
+  - /cloud/cloud-resource-types/?per_page=4&page=5
+  cloud:cloudresourcetype_notes:
+  - /cloud/cloud-resource-types/de3a66de-bf98-4f53-89e9-4be79ee0632d/notes/
+  - /cloud/cloud-resource-types/17a82f09-4da4-4543-a3be-7a4678bd694a/notes/
+  cloud:cloudservice:
+  - /cloud/cloud-services/1a157c39-2e74-4ac4-9c41-b3293fca7ea9/
+  - /cloud/cloud-services/9a1d438b-0546-4e0a-888b-6a8d83a89069/
+  cloud:cloudservice_changelog:
+  - /cloud/cloud-services/1a157c39-2e74-4ac4-9c41-b3293fca7ea9/changelog/
+  - /cloud/cloud-services/9a1d438b-0546-4e0a-888b-6a8d83a89069/changelog/
+  cloud:cloudservice_list:
+  - /cloud/cloud-services/
+  - /cloud/cloud-services/?per_page=4&page=5
+  cloud:cloudservice_notes:
+  - /cloud/cloud-services/1a157c39-2e74-4ac4-9c41-b3293fca7ea9/notes/
+  - /cloud/cloud-services/9a1d438b-0546-4e0a-888b-6a8d83a89069/notes/
+  core-api:api-root:
+  - /api/core/
+  - /api/core/
+  - /api/core/<drf_format_suffix:format>
+  - /api/core/
+  - /api/core/<drf_format_suffix:format>
+  - /api/core/
+  - /api/core/<drf_format_suffix:format>
+  - /api/core/
+  - /api/core/<drf_format_suffix:format>
+  - /api/core/
+  - /api/core/<drf_format_suffix:format>
+  - /api/core/
+  - /api/core/<drf_format_suffix:format>
+  - /api/core/
+  - /api/core/<drf_format_suffix:format>
+  - /api/core/
+  - /api/core/<drf_format_suffix:format>
+  - /api/core/
+  - /api/core/<drf_format_suffix:format>
+  - /api/core/
+  - /api/core/<drf_format_suffix:format>
+  core-api:api-status:
+  - /api/core/status/
+  core-api:csv-import-fields:
+  - /api/core/csv-import-fields/
+  - /api/core/csv-import-fields/
+  core-api:filtersetfield-list-lookupchoices:
+  - /api/core/filterset-fields/lookup-choices/
+  - /api/core/filterset-fields/lookup-choices/
+  core-api:filtersetfield-retrieve-lookupvaluedomelement:
+  - /api/core/filterset-fields/lookup-value-dom-element/
+  - /api/core/filterset-fields/lookup-value-dom-element/
+  core-api:graphql-api:
+  - /api/core/graphql/
+  core:about:
+  - /core/about/
+  core:graphql:
+  - /core/graphql/
+  core:home:
+  - /core/
+  core:metrics:
+  - /core/metrics/
+  core:search:
+  - /core/search/
```

Comment:
Do these actually have `/core/` in the URLs? I don't believe so. Same question on at least some of the `core-api` URLs above.